style.css

```diff
@@ -645,6 +721,21 @@ h4 {
   margin-left: 15px;
 }
 
+#form-id #errormessage {
+  position: absolute;
```

Comment:
Well done on the error message 👏🏽 but this line is making your error not appear near the form when the email is not fully lowercase, you can either remove this and have it `hidden` by default and toggle it using JavaScript DOM interactions. 🤓 Check my other comment.